app/src/main/java/org/dhis2/usescases/programStageSelection/ProgramStageSelectionActivity.kt

```diff
@@ -26,6 +27,9 @@ class ProgramStageSelectionActivity : ActivityGlobalAbstract(), ProgramStageSele
     @Inject
     lateinit var presenter: ProgramStageSelectionPresenter
 
+    @Inject
+    lateinit var resourceManager: ResourceManager
```

Comment:
What do you think about injecting the `EventResourceProvider` to reduce duplicated code?